storage/innobase/row/row0mysql.cc

```diff
@@ -2179,6 +2260,20 @@ row_update_for_mysql(row_prebuilt_t* prebuilt)
 	DBUG_RETURN(err);
 }
 
+/** Does an update or delete of a row for MySQL.
+@param[in]	mysql_rec	row in the MySQL format
+@param[in,out]	prebuilt	prebuilt struct in MySQL handle
+@return error code or DB_SUCCESS */
+dberr_t
+row_update_for_mysql(
+	row_prebuilt_t*		prebuilt,
+	bool			vers_set_fields)
+{
+	ut_a(prebuilt->template_type == ROW_MYSQL_WHOLE_ROW);
+	return (row_update_for_mysql_using_upd_graph(
+		prebuilt, vers_set_fields));
+}
+
```

Comment:
This wrapper function is unnecessary. I probably had removed it at some point after the original code was cloned.

Reply:
https://github.com/tempesta-tech/mariadb/issues/305